datashader/datatypes.py

```diff
@@ -123,17 +123,26 @@ class RaggedDtype(ExtensionDtype):
 
     @property
     def name(self):
+        """
+        See docstring for ExtensionDtype.name
+        """
```

Comment:
With most docs/API tools, docstrings will simply be inherited as-is if you don't specify one here, so please remove these altogether unless they need to say something explicitly about how they relate to the parent class.  You can mention the parent class explicitly in the class docstring, once, with something like "Methods not otherwise documented here are inherited from ExtensionDtype; please see the corresponding method on that class for the docstring".